docker/moj.json

```diff
@@ -22,7 +22,7 @@
     "ministryofjustice/dw-document-revisions":"dev-master",
     "ministryofjustice/dw-live-drafts":"dev-composer",
     "ministryofjustice/dw-markdown":"dev-composer",
-    "ministryofjustice/intranet-theme-clarity":"dev-composer",
+    "ministryofjustice/intranet-theme-clarity":"dev-master",
```

Comment:
` Problem 1
wp_local_dev   |     - The requested package ministryofjustice/intranet-theme-clarity dev-master exists as ministryofjustice/intranet-theme-clarity[dev-composer] but these are rejected by your constraint.
`